docs/sources/fqdn-templating.md

```diff
@@ -0,0 +1,280 @@
+# FQDN Templating Guide
```

Comment:
It's not really a source, like the other ones.

Wdyt about put this file in "advanced topics" section of the doc ?

Reply:
Make sense. Created folder `advanced` and moved few things there.